cmd/operator-sdk/generate/csv.go

```diff
@@ -53,35 +55,58 @@ for the operator. This file is used to publish the operator to the OLM Catalog.
 
 A CSV semantic version is supplied via the --csv-version flag. If your operator
 has already generated a CSV manifest you want to use as a base, supply its
-version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.
+version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.`,
+		Example: `
+TODO
+	`,
 
-Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config.yaml`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// The CSV generator assumes that the deploy and pkg directories are
-			// present at runtime, so this command must be run in a project's root.
-			projutil.MustInProjectRoot()
-
 			if len(args) != 0 {
 				return fmt.Errorf("command %s doesn't accept any arguments", cmd.CommandPath())
 			}
 			if err := c.validate(); err != nil {
 				return fmt.Errorf("error validating command flags: %v", err)
 			}
+			// Default for crd dir if unset
+			if c.crdDir == "" {
+				c.crdDir = c.deployDir
+			}
 			if err := c.run(); err != nil {
 				log.Fatal(err)
 			}
 			return nil
 		},
 	}
 
-	cmd.Flags().StringVar(&c.csvVersion, "csv-version", "", "Semantic version of the CSV")
+	cmd.Flags().StringVar(&c.csvVersion, "csv-version", "",
+		"Semantic version of the CSV")
 	if err := cmd.MarkFlagRequired("csv-version"); err != nil {
 		log.Fatalf("Failed to mark `csv-version` flag for `generate csv` subcommand as required: %v", err)
 	}
 	cmd.Flags().StringVar(&c.fromVersion, "from-version", "",
 		"Semantic version of an existing CSV to use as a base")
-	cmd.Flags().StringVar(&c.csvConfigPath, "csv-config", "",
-		"Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml")
+
+	cmd.Flags().StringVar(&c.deployDir, "deploy-dir", "deploy",
+		`Project relative path to root directory for operator manifests (Deployment, RBAC, CRDs).
```

Comment:
Allows relative and full path. So, wdyt about something such as `Operator Manifest directory path where are the Deployment, RBAC and CRDs. (E.g deploy/) (allows relative paths from to root project directory).`

Note that thee root word his shows not necessary.